rasa/core/tracker_store.py

```diff
@@ -720,6 +723,95 @@ def keys(self) -> Iterable[Text]:
         return [c["sender_id"] for c in self.conversations.find()]
 
 
+class FirebaseRealtimeTrackerStore(TrackerStore):
+    """Stores conversation history in Firebase Realtime Database
+
+    Args:
+        db_root : root branch name to store conversations
+        firebase_key_file_path : path to firebase admin sdk key file
+
+    """
+
+    def __init__(
+        self,
+        domain: Domain,
+        firebase_key_file_path: Text,
+        db_storage_path: Text,
+        database_url: Text,
+        event_broker: Optional[EndpointConfig] = None,
+        **kwargs: Dict[Text, Any],
+    ) -> None:
+
+        self.store = {}
+        self.key_file_path = firebase_key_file_path
+        self.database_url = database_url
+        self.db_storage_path = db_storage_path
+
+        self.cred = credentials.Certificate(self.key_file_path)
+        init_app = firebase_admin.initialize_app(
```

Comment:
I would store this as an instance attribute:
```suggestion
        self.db = firebase_admin.initialize_app(
```